TA Signals/TempMainView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TempMainView.swift
+//  TA Signals
+//
+//  Created by Gabriel Maldonado Almendra on 27/7/21.
+//
+
+import SwiftUI
+
+struct TempMainView: View {
+    
+    // 1. The fetch property will observe the FetchToDo class for changes
+    @State var fetchedObject = StocksDataFetcher()
+    @State var showFetchDetails : Bool = false
+    @State var isMarketOpen : Bool = true
+    @State var openOrClosed : Bool = false
```

Comment:
`@State var openOrClosed : Bool = false`

I am confused by this variable name.  When you write out a method name or a variable name you want to try and make it make sense as if a person was reading it.  This is something I really struggle with, and is certainly subjective to the reader, but think about your if else below

` if !openOrClosed {`

if it is not open or closed?  Verbally this implies there is a third option that is not open and is not closed.  Compare it to the variable just before:

`isMarketOpen : Bool = true`

if the market is open then do something, else don't.  the naming follows what you are logically doing with your code